packages/core-plugin-api/api-report.md

```diff
@@ -424,7 +424,8 @@ export type ExternalRouteRef<
 // @public
 export type FeatureFlag = {
   name: string;
-  pluginId: string;
+  pluginId?: string;
```

Comment:
Hmm this unfortunately would be a breaking change, I think previously we have used `pluginId: ''` for anything registered at the `app` level.

Reply:
It's ugly, but until we're ready to make this breaking change we should just use the workaround for now.